src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -296,13 +296,13 @@ public static function createFromGlobals()
      * The information contained in the URI always take precedence
      * over the other information (server and parameters).
      *
-     * @param string $uri        The URI
-     * @param string $method     The HTTP method
-     * @param array  $parameters The query (GET) or request (POST) parameters
-     * @param array  $cookies    The request cookies ($_COOKIE)
-     * @param array  $files      The request files ($_FILES)
-     * @param array  $server     The server parameters ($_SERVER)
-     * @param string $content    The raw body data
+     * @param string      $uri        The URI
+     * @param string      $method     The HTTP method
+     * @param array       $parameters The query (GET) or request (POST) parameters
+     * @param array       $cookies    The request cookies ($_COOKIE)
+     * @param array       $files      The request files ($_FILES)
+     * @param array       $server     The server parameters ($_SERVER)
+     * @param string|null $content    The raw body data
```

Comment:
it affects not only this method, but other methods and property as well

Reply:
@keradus what do you mean? Looking at the code, passing null is not supported for the other argument. Yes, technically you can, but then, the behavior is certainly not part of any contract - except for $content.

Reply:
Oups, I needed to post to actually understand your comment. Other *methods* :)
@plazma0 would you mind reviewing the `$content` argument then?
Please note that this PR should be merged against branch 2.7, so that if you know how to do it, you're welcomed.